packages/cli/src/report/report-generator.spec.ts

```diff
@@ -1,14 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import 'reflect-metadata';
-
 import { AxeReportParameters, Report, Reporter, ReporterFactory, SummaryScanResults } from 'accessibility-insights-report';
 import { AxeResults } from 'axe-core';
 import * as MockDate from 'mockdate';
+import 'reflect-metadata';
```

Comment:
this may break test run; we need `reflect-metadata` to be first import

Reply:
Gotcha! Will update.